src/plugins/intel_cpu/src/nodes/priorbox.cpp

```diff
@@ -220,8 +220,8 @@ void PriorBox::execute(const dnnl::stream& strm) {
                         float box_height_ratio = fixed_size[s] * 0.5f / ar;
                         for (int64_t r = 0; r < density_; ++r) {
                             for (int64_t c = 0; c < density_; ++c) {
-                                float center_x_temp = center_x - fixed_size_ / 2 + shift / 2.f + c * shift;
-                                float center_y_temp = center_y - fixed_size_ / 2 + shift / 2.f + r * shift;
+                                float center_x_temp = center_x - fixed_size_ / 2.f + shift / 2.f + c * shift;
```

Comment:
what check is it?

Reply:
Discussed offline, it is [bugprone-integer-division](https://clang.llvm.org/extra/clang-tidy/checks/bugprone/integer-division.html)